src/connectors/numeric-menu/connectNumericMenu.ts

```diff
@@ -126,25 +133,64 @@ const connectNumericMenu: NumericMenuConnector = function connectNumericMenu(
       }));
 
     const connectorState: ConnectorState = {};
+    let sendEvent;
```

Comment:
if you put it in connectorState higher, you probably don't need the variable

Reply:
Do you mean `connectorState.sendEvent`?

Reply:
yes, or is that something else?

Reply:
https://github.com/algolia/instantsearch.js/commit/ce4f5f0840dcbb893066e0ff40309b1f4175ea2b#diff-5a213002ed9abd35556c3f655cc88ef6R162

Now that connectNumericMenu already has `connectorState`, I put `sendEvent` inside it for consistency.